seed/photosSeeder.js

```diff
@@ -1,42 +1,129 @@
-    'Authorization': 
-    'Client-ID 216bd408c1b2811738a6fa3f3e534ec60328f83037df125a7d897283196d99e5'
-  }
-})
 
 
 // // generate a random photosCount that represents the number of photos to generate for a particular restaurant
 
-// var generateRandomPhotoCountForRestaurant = function(min, max) {
-//   return Math.floor((Math.random() * (max - min + 1)) + min);
-// }
+var generateRandomPhotoCountForRestaurant = function(min, max) {
+  return Math.floor((Math.random() * (max - min + 1)) + min);
+}
+
+const photoCount = generateRandomPhotoCountForRestaurant(1, 25);
+
+
+
+
+// const collectionId = 622212; // the collection id
+
+// const heroImageWidth = 694;
+// const heroImageHeight = 230;
+
+// const leftTwoGalleryWidthDimension = 143.83;
+
+// const centerGalleryDimension = 289.69;
+
+// const rightSixGalleryDimension = 95.22;
+
+// const galleryScrollerDimension = 400;
+
+// const photoGalleryPaneWidth = 610
+// const photoGalleryPaneHeight = 270
+
+
+
+// `https://source.unsplash.com/collection/${x}/${heroImageWidth}x${heroImageHeight}/?sig=${generateRandomPhotoCountForRestaurant(1, collectionNumber)}`
+
+// `https://source.unsplash.com/collection/${collectionId}/${leftTwoGalleryWidthDimension}x${leftTwoGalleryWidthDimension}/?sig=${generateRandomPhotoCountForRestaurant(1, collectionNumber)}`
+
+// `https://source.unsplash.com/collection/${collectionId}/${centerGalleryDimension}x${centerGalleryDimension}/?sig=${generateRandomPhotoCountForRestaurant(1, collectionNumber)}`
+
+// `https://source.unsplash.com/collection/${collectionId}/${rightSixGalleryDimension}x${rightSixGalleryDimension}/?sig=${generateRandomPhotoCountForRestaurant(1, collectionNumber)}`
+
+// `https://source.unsplash.com/collection/${collectionId}/${galleryScrollerDimension}x${galleryScrollerDimension}/?sig=${generateRandomPhotoCountForRestaurant(1, collectionNumber)}`
+
 
 
 
-// console.log(generateRandomPhotoCountForRestaurant(0, 25));
+`https://source.unsplash.com/collection/2080789/${heroImageWidth}x${heroImageHeight}/sig=${5}`
 
-// "urls": {
-//   "raw": ""https://images.unsplash.com/photo-1525159831892-59d292d558b4?ixlib=rb-1.2.1&q=85&fm=jpg&crop=entropy&cs=srgb&ixid=eyJhcHBfaWQiOjU0Njg1fQ"",
-//   "full": "https://hd.unsplash.com/photo-1416339306562-f3d12fefd36f",
-//   "regular": 
-// "https://images.unsplash.com/photo-1416339306562-f3d12fefd36f?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max&s=92f3e02f63678acc8416d044e189f515",
-// "small": 
-// "https://images.unsplash.com/photo-1416339306562-f3d12fefd36f?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&s=263af33585f9d32af39d165b000845eb",
-// "thumb": 
-// "https://images.unsplash.com/photo-1416339306562-f3d12fefd36f?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=200&fit=max&s=8aae34cf35df31a592f0bef16e6342ef"
-// };
+`https://source.unsplash.com/collection/2080789/400x400/sig=${6}`
+// `https://source.unsplash.com/collection/4239193/400x400`
```

Comment:
there are duplicates, I needed move away from using variable size dimensions with string literals inside the url because it changed the outcome of the photo that the unsplash api would send back, thanks for the comment, miss you guys.